frontend/src/views/camp/NavigationCamp.vue

```diff
@@ -7,23 +7,23 @@
              text
              width="216">
         <v-toolbar-title>
-          {{ camp().title | loading('Camp wird geladen…') }}
+          {{ camp().title | loading('Lager wird geladen…') }}
```

Comment:
You can use `$tc` in here to translate this as well:
`loading($tc('some.translation.key'))`